lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -557,6 +557,28 @@ html.video-fullscreen {
       }
     }
 
+    section.xqa-modal {
```

Comment:
all of these modals have the same values.  This can be combined

`section.xqa-modal, section.staff-modal, section.history-modal { ...`